CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## 15.5.4
+* Fix UMD Build. ([@acdlite](https://github.com/acdlite) in [31e9344](https://github.com/reactjs/prop-types/commit/31e9344ca3233159928da66295da17dad82db1a8))
```

Comment:
There is also an important change in 15.4.4:

* Remove the accidentally included typechecking code from the production build.